test/tc_webhook_config_test.go

```diff
@@ -97,3 +97,19 @@ func (e *e2e) getWebhookAttribute(hook, attr string) string {
 		"spo-mutating-webhook-configuration",
 		"--output", "jsonpath="+jsonPath)
 }
+
+func (e *e2e) testCaseWebhookHTTP([]string) {
+	if !e.testWebhookHTTP {
+		e.T().Skip("Skipping webhook HTTP version related tests")
+	}
+	e.logf("Test webhook HTTP version")
+
+	endpoints := []string{
+		curlHeaderCMD + metricsURL + "/metrics",
+		curlHeaderCMD + metricsURL + "/metrics-spod",
```

Comment:
you should remove the `/` here

Reply:
Hmm, weird. Testing manually on cluster the extra slash didn't seem to make a difference.
But I can try and see how CI behaves.